Source/WebCore/platform/network/BlobResourceHandle.cpp

```diff
@@ -224,16 +224,18 @@ void BlobResourceHandle::doStart()
 void BlobResourceHandle::getSizeForNext()
 {
     ASSERT(isMainThread());
+    Ref protectedThis { *this };
```

Comment:
It is a little odd to add protectedThis here since this is a regular method.
Should we move it to BlobResourceHandle::didGetSize where it would make more sense given it is a Client API, like done in NetworkDataTaskBlob?

Reply:
Ok